compute/instances/troubleshooting/bootscript-eol.mdx

```diff
@@ -0,0 +1,90 @@
+
+Certain older legacy Instance types (e.g. **VC1-x**, **X64-x**, **Start1-x**, and **Stardust**) offered the default option to use a bootscript (a preconfigured boot configuration) to start your Instance. This option has become deprecated and will no longer be supported.
+If you are still using one of these Instance types with a bootscript, you have to change your boot configuration in order to keep your Instance operational.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created your SSH key and added it to your account](/console/my-project/how-to/create-ssh-key/)
+  - You have [an Instance](/compute/instances/how-to/create-an-instance) using a bootscript
+</Message>
+
+## Migration options for Instances using bootscripts
+
+<Message type="important">
+  Local volume(s) that do not support UEFI cannot be used to boot any other type of Instance. You must retrieve and manually migrate your data to an Instance that supports UEFI boot.  
+  Unfortunately, we cannot identify, on the Scaleway side, which customers are using OS images with or without UEFI partitions. The user themselves can only identify this.   
+  To do so, [connect to your Instance](/compute/instances/how-to/connect-to-instance) using SSH and run the following command to list the EFI directories:
+  ```
+  ls -ld /sys/firmware/efi
+  ```
+  If you can see the following output, your Instance uses UEFI boot:
+  ```
+  root@my-virtual-instance:~# ls -ld /sys/firmware/efi
+  drwxr-xr-x 6 root root 0 Feb 20 11:16 /sys/firmware/EFI
+  ```
+  In this case use **option 4** to migrate your data.    
+  If you see `ls: cannot access /sys/firmware/efi: No such file or directory` the UEFI directories are not present on your Instance. Migrate your data using **option 1, 2 or 3**.
+</Message>
+
+### Instances using bootscript method and an old OS image without UEFI
+
+Most of the existing **VC1-x**, **X64-x**, **Start1-x**, and **Stardust** Instances fall into this category.
+
+#### Option 1: Create a unified snapshot of the volume(s) and rebuild block volume(s) from the snapshot
+
+1. [Create a snapshot](/compute/instances/how-to/create-a-snapshot/) of the volume using the **unified** snapshot type.
+2. [Create a new block volume](/storage/block/how-to/create-a-volume) from the previous unified snapshot.
+3. [Create and start](/compute/instances/how-to/create-an-instance/) a new Instance using its own boot volume in local boot mode.
+4. [Attach](/storage/block/how-to/attach-a-volume/) the block volume to the new Instance.
+5. Once logged into the Instance, [mount the partition](/storage/block/how-to/mount-and-use-volume/) available in the block volume and access your data.
+6. Delete the old Instance that was using a bootscript once the migration is completed.
+
+#### Option 2: Create a unified snapshot of the volume(s) and export it to S3 to retrieve the data
```

Comment:
Here in the title, the snapshot type doesn't need to be `unified` (the first line of the instructions reflect that)

Reply:
```suggestion
#### Option 2: Create a snapshot of the volume(s) and export it to S3 to retrieve the data
```